apps/ensadmin/src/app/@actions/api/subgraph/page.tsx

```diff
@@ -1,10 +1,12 @@
 "use client";
 
 import { CopyButton } from "@/components/copy-button";
-import { useActiveENSNodeUrl } from "@/hooks/active/use-active-ensnode-url";
+import { useSelectedENSNodeConnection } from "@/hooks/active/use-selected-ensnode-connection";
 
 export default function ActionsSubgraphCompatPage() {
-  const baseUrl = useActiveENSNodeUrl();
+  // TODO: we need a broader refactor to recognize the difference between
+  // a selected connection being in a valid format or not.
+  const baseUrl = useSelectedENSNodeConnection().rawSelectedConnection;
```

Comment:
```suggestion
  const { rawSelectedConnection : baseUrl} = useSelectedENSNodeConnection();
```

@lightwalker-eth is there any reason we can't format it like this how we do other hook usage?